package.json

```diff
@@ -15,7 +15,9 @@
     "registry": "https://artifacts.aue1d.saasure.com/artifactory/api/npm/npm-okta"
   },
   "scripts": {
-    "lint:report": "grunt jshint",
+    "lint": "eslint ./lib ./test",
+    "lint:fix": "eslint --fix ./lib ./test",
+    "lint:report": "eslint -f checkstyle -o build2/loginjs-checkstyle-result.xml ./lib ./test",
```

Comment:
Gotcha. I just copied it from the old Gruntfile.js, so I figured that's what we did elsewhere. I'll change it